machinery/objects_test.go

```
@@ -1259,8 +1251,8 @@ func TestObjectEngine_Teardown(t *testing.T) {
 						*obj = *actualObject
 					}).
 					Return(nil)
-				writer.On("Delete", mock.Anything, mock.Anything, mock.Anything).
-					Panic("Delete should not be called on the engine writer")
+				// writer.On("Delete", mock.Anything, mock.Anything, mock.Anything).
```

golangci-lint (2.12.2)

[error] Duplicate words (mock.Anything,) found

(dupword)